Getting Error in Usage of swisstable

I'm getting the following error:

```
../../go/pkg/mod/github.com/thepudds/swisstable@v0.0.0-20221011152303-9c77dc657777/match_stub.go:5:6: missing function body
```

Whenever I try to run the tests.

I took a peek at the package "thepudds/swisstable" and I noticed this file:

match_amd64.s

which looks like it should be providing the stub for AMD64 - but doesn't seem to be.  I am running on a Mac with AMD64.

In regards to what I've tried - before I tried running the tests - I ran the following:

```
go mod tidy
```

which did install the following:

```
go: downloading github.com/thepudds/fzgen v0.4.2
go: downloading gopkg.in/check.v1 v0.0.0-20161208181325-20d25e280405
go: downloading github.com/sanity-io/litter v1.5.1
```

Then when I tried to run the tests and I got the same error - I also ran the following:

```
go install
```

I did check out the README.md here:
https://github.com/thepudds/swisstable

But, didn't see anything with regards to special instructions for AMD64.

Thanks!

--- a/go.mod
+++ b/go.mod
@@ -5,7 +5,6 @@ go 1.18
 require (
 	github.com/dolthub/maphash v0.0.0-20221220182448-74e1e1ea1577
 	github.com/stretchr/testify v1.8.1
-	github.com/thepudds/swisstable v0.0.0-20221011152303-9c77dc657777
 )
 
 require (

--- a/map_bench_test.go
+++ b/map_bench_test.go
@@ -22,8 +22,6 @@ import (
 
 	"github.com/stretchr/testify/require"
 
-	"github.com/thepudds/swisstable"
-
 	"github.com/stretchr/testify/assert"
 )
 
@@ -51,9 +49,6 @@ func BenchmarkInt64Maps(b *testing.B) {
 			})
 			b.Run("swiss.Map", func(b *testing.B) {
 				benchmarkSwissMap(b, generateInt64Data(n))
-			})
-			b.Run("swisstable.Map", func(b *testing.B) {
-				benchmarkThepuddsMap(b, generateInt64Data(n))
 			})
 		})
 	}
@@ -111,24 +106,6 @@ func benchmarkSwissMap[K comparable](b *testing.B, keys []K) {
 	}
 	assert.True(b, ok)
 	b.ReportAllocs()
-}
-
-func benchmarkThepuddsMap(b *testing.B, keys []int64) {
-	n := uint32(len(keys))
-	mod := n - 1 // power of 2 fast modulus
-	require.Equal(b, 1, bits.OnesCount32(n))
-	m := swisstable.New(len(keys))
-	for _, k := range keys {
-		m.Set(swisstable.Key(k), swisstable.Value(k))
-	}
-	b.ResetTimer()
-	var v swisstable.Value
-	var ok bool
-	for i := 0; i < b.N; i++ {
-		v, ok = m.Get(swisstable.Key(keys[uint32(i)&mod]))
-	}
-	assert.NotNil(b, v)
-	assert.True(b, ok)
 }
 
 func generateInt64Data(n int) (data []int64) {

